MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -169,13 +169,15 @@ class RouteManeuverViewController: UIViewController {
         }
     }
     
-    func updateStreetNameForStep() {
+    func updateStreetNameForStep(currentLeg: RouteLeg? = nil) {
```

Comment:
On second thought, since `step` is a property that [gets set](https://github.com/mapbox/mapbox-navigation-ios/blob/ff6f9e56ac213991e517d7eb3c13f3733bc8b558/MapboxNavigation/RoutePageViewController.swift#L51) when the view controller is set up, maybe `leg` should be a property too? That would eliminate the need to pass it in here.